src/popup/utils/constants.ts

```diff
@@ -361,8 +364,10 @@ export const CURRENCIES: ICurrency[] = [
   },
 ];
 
-export const FUNCTION_TYPE_DEX: Record<'pool' | 'swap' | 'allowance' | 'maxSpent' | 'minReceived', TxFunctionRaw[]> = {
+export const FUNCTION_TYPE_DEX: Record<'pool' | 'addLiquidity' | 'removeLiquidity' |'swap' | 'allowance' | 'maxSpent' | 'minReceived', TxFunctionRaw[]> = {
   pool: ['remove_liquidity', 'remove_liquidity_ae', 'add_liquidity', 'add_liquidity_ae'],
+  removeLiquidity: ['remove_liquidity', 'remove_liquidity_ae'],
+  addLiquidity: ['add_liquidity', 'add_liquidity_ae'],
```

Comment:
are you sure that addLiquidity and removeLiquidity should be part of DEX functions? @CedrikNikita

Reply:
In general, they are not a part of DEX, but we were showing them as part of it, since that was only case for our wallet.